src/EPR.Accreditation.Facade/Controllers/OverseasSiteController.cs

```diff
@@ -40,5 +44,32 @@ public OverseasSiteController(IOverseasSiteService overseasSiteService)
 
             return Ok();
         }
+
+        [HttpGet("Outputs")]
+        [ProducesResponseType(typeof(OverseasReprocessingSiteOutputs), 200)]
+        public async Task<IActionResult> GetOverseasSiteOutputs(Guid accreditationExternalId, Guid siteExternalId)
+        {
+            var dto = await _accreditationService.GetOverseasReprocessingSiteOutputs(accreditationExternalId, siteExternalId);
+
+            if (dto == null)
+                return NotFound();
```

Comment:
Again, defra coding standards are to have even single line if statements have the braces